src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -630,6 +630,15 @@ static Option captureCallState(String... capturedState) {
          *     <li>WSAGetLastError</li>
          *     <li>errno</li>
          * </ul>
+         * The following snipet shows how to obtain the names of the supported captured value layouts:
+         * {@snippet lang = java:
+         *    String capturedNames = Linker.Option.captureStateLayout().memberLayouts().stream()
+         *        .map(MemoryLayout::name)
+         *        .filter(Optional::isPresent)
+         *        .map(Optional::get)
```

Comment:
[`Optional::stream()`] was added specifically for this:
```suggestion
         *        .flatMap(Optional::stream)
```

[`Optional::stream()`]: https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/util/Optional.html#stream()